azure-kusto-ingest/tests/test_ingest_telemetry.py

```diff
@@ -0,0 +1,26 @@
+from io import BytesIO
+
+from azure.kusto.ingest import FileDescriptor, BlobDescriptor, StreamDescriptor
+from azure.kusto.ingest.ingestion_properties import IngestionProperties
+
+
+def test_get_tracing_attributes():
```

Comment:
Didn't we talk about adding a specific test here, similar to test_create_http_attributes?

Reply:
yes, this test uses double inclusion to prove that the keys of the attributes are equal to the dummy key attributes above.